### ISSUE TITLE
queries with -- in fron to fcolumn names

### ISSUE THREAD

**Issue by taringoo** [macbre/sql-metadata#147]
it seems the parser cannot parse anything after -- in front of any item in the query

Parser('select --k, n from m')

returns empty table list.

**Comment by collerek** [comment]
Yes, as this IS sql inline comment so it behaves correctly: https://www.w3schools.com/sql/sql_comments.asp

**Comment by Oremite** [comment]
This is commenting out everything after the in-line comments:

```
simple_sql = """select *
from foo -- this comment is hiding the whole rest of the query
join bar using(whatever)
where I = 'am'"""

p = Parser(simple_sql)
print(p.tables)
```
returns _['foo']_, and ignores bar.

Interestingly enough, the "generalize" seems to work. If you continue the previous code:
```
p2 = Parser(p.generalize)
print(p2.tables)
```
returns _['foo', 'bar']_

**Comment by Oremite** [comment]
This should be renamed to "Queries with in-line comments (--) are commenting out the entire rest of the query."

**Comment by collerek** [comment]
Ok, we are removing new lines in pre processing now i get what you mean. Thanks for reporting! 🙂
